code/game/objects/items/cards_ids.dm

```diff
@@ -633,9 +634,6 @@
 /obj/item/card/id/proc/alt_click_can_use_id(mob/living/user)
 	if(!isliving(user))
 		return FALSE
-	if(!user.can_perform_action(src, FORBID_TELEKINESIS_REACH))
-		return FALSE
-
 	return TRUE
```

Comment:
this abstraction is kinda silly at this point. it could be simplified in click_alt_secondary with just `!isliving(user)`

Reply:
This proc is actually overriden by subtypes. One example here
https://github.com/tgstation/tgstation/blob/0678c610ab36b414b26ee982830e8444aedfc99e/code/game/objects/items/cards_ids.dm#L1242